feconf.py

```diff
@@ -61,8 +61,14 @@
     FRONTEND_TEMPLATES_DIR = os.path.join('core', 'templates', 'dev', 'head')
 DEPENDENCIES_TEMPLATES_DIR = (
     os.path.join(EXTENSIONS_DIR_PREFIX, 'extensions', 'dependencies'))
+
+VALUE_GENERATORS_DIR_FOR_JS = os.path.join(
+    'local_compiled_js', 'extensions', 'value_generators')
 VALUE_GENERATORS_DIR = os.path.join('extensions', 'value_generators')
-VISUALIZATIONS_DIR = os.path.join('extensions', 'visualizations')
+
+VISUALIZATIONS_DIR = os.path.join(
```

Comment:
For consistency, should this be renamed VISUALIZATIONS_DIR_FOR_JS?

(Also, are non-JS files in extensions/visualizations not used?)

Reply:
No other files are not used in `visualization_registry.py`. The HTML templates are used in the directives itself. 

Done.